File: src/lolicon.cs
using System;
using System.Diagnostics;

class MainClass {
  public static void Main (string[] args) {
    Console.WriteLine("What\'s the girl\'s age?");
    int age = Convert.ToInt32(Console.ReadLine());
    CallTheFbi(age);
  }
  static void CallTheFbi(int age) {
    if (age < 15) {
      System.Diagnostics.Process.Start("call_911");
    } else {
      Console.WriteLine("Probably just something weird happened");
    }
  }
}

File: src/lolicon.vb
Imports System

Module Lolicon

    Sub Intice()
        ' Nothing because weebs are already attracted To underage anime girls
    End Sub

    Sub CallTheFbi(age As Integer)
        If age < 15 Then
            Process.Start("call_911")
        Else
            Console.WriteLine("Probably just something weird happened.")
        End If
    End Sub

    Sub Main(args As String())
        Console.WriteLine("What's the girl's age?")
        Dim age As Integer
        age = Convert.ToInt32(Console.ReadLine())
        CallTheFbi(age)
    End Sub

End Module
